src/test/java/io/getstream/chat/java/ChannelTest.java

```diff
@@ -121,6 +122,27 @@ void whenTruncateChannel_thenNoException() {
                 .request());
   }
 
+  @DisplayName("Can truncate channel with options")
+  @Test
+  void whenTruncateChannelWithOptions_thenNoException() {
+    // We should not use testChannel to not remove testMessage
+    ChannelGetResponse channelGetResponse =
+        Assertions.assertDoesNotThrow(() -> createRandomChannel());
+    MessageRequestObject message =
+        MessageRequestObject.builder()
+            .text("Truncate channel")
+            .userId(testUserRequestObject.getId())
+            .build();
+    Assertions.assertDoesNotThrow(
+        () ->
+            Channel.truncate(
```

Comment:
Can we check that we have message in the response ?

Reply:
ya sure